influence_benchmark/experiments/slurm/kickoff_slurm.sh

```diff
@@ -3,15 +3,16 @@
 ###############################################################
 # PARAMETERS
 
-CONFIG_NAMES="KTO_weak_therapist1t_env" # Space-separated list of config names
+CONFIG_NAMES="KTO_therapist" # Space-separated list of config names
 
 # SLURM job parameters
-SLURM_CPUS_PER_TASK=8
-SLURM_MEM="64gb"
+SLURM_CPUS_PER_TASK=16
+SLURM_MEM="200gb"
```

Comment:
Why did you increase this? Do we need it for some runs?

Reply:
for the bigger models we need more memory i think, I don't usually commit my kickoff_slurm but i wanted to add a comment